### PR TITLE
Fix errors and omissions in the README.md file

### PR DESCRIPTION
**Pull request by markdryan**
Fix the following issues:

1. The description of workloads was very out of date.
2. The inherits feature and the run command were not documented.
3. The use of the ccloudvm command was inconsistent.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -203,24 +220,29 @@ fields are currently defined:
...
+- vm		  : Contains information about creating and booting the instance.
 
-### The Instance Data Document
+The vm field supports a number of child fields.
 
-The second document is called the instance data document.  It contains information that is
-set when an instance is first created but may be modified at a later date, without deleting
-the instance.    Four fields are currently defined:
+- mem_mib    : Number of mebibytes to assign to the VM.  Defaults to 1024 MiBs.
+- disk_gib   : Number of gebibytes to assign to the rootfs of the VM.  Defaults to 60 GiB.
```
Comment (rbradford) [inline README.md]: s/gebibytes/gibibytes/

File: README.md [hunk 2]
```diff
@@ -203,24 +220,29 @@ fields are currently defined:
...
-### The Instance Data Document
+The vm field supports a number of child fields.
 
-The second document is called the instance data document.  It contains information that is
-set when an instance is first created but may be modified at a later date, without deleting
-the instance.    Four fields are currently defined:
+- mem_mib    : Number of mebibytes to assign to the VM.  Defaults to 1024 MiBs.
+- disk_gib   : Number of gebibytes to assign to the rootfs of the VM.  Defaults to 60 GiB. 
+- cpus	     : Number of CPUs to assign to the VM.  Defaults to 1 VCPU.
+- ports	     : Sequence of port objects which map host ports to guest ports
```
Comment (rbradford) [inline README.md]: Indentation?

File: README.md [hunk 3]
```diff
@@ -203,24 +220,29 @@ fields are currently defined:
...
+- drives     : Sequence of drive objects which identify resources accessible on the host to be made available as block devices on the guest.
 
-- mem_mib : Number of mebibytes to assign to the VM.  Defaults to half the memory on your machine
-- cpus    : Number of CPUs to assign to the VM.  Defaults to half the cores on your machine
-- ports   : Slice of port objects which map host ports on 127.0.0.1 to guest ports
-- mounts  : Slice of mount objects which describe the folders shared between the host and the guest
-- drives  : Slice of drive objects which identify resources accessible on the host to be made available as block devices on the guest.
+Each instance is associated with one of the host's IP addresses.  Only one instance can be
+associated with one host IP address at any one time.  The user can specify which host IP
+address to use for an instance when the instance is created using the -hostip option.
```
Comment (rbradford) [inline README.md]: --hostip

**Comment by markdryan** [comment]
Thanks.   All issues have been fixed.

**Pull request merged**